### ISSUE TITLE
The configuration of the 8i dataset.

### ISSUE THREAD

**Issue by GoodJuicee** [aniqueakhtar/PointCloudUpsampling#7]
Hello, how to set up the 8i data in train.py?

**Comment by aniqueakhtar** [comment]
Please run the preprocessing in this file to generate the dataset:
`pre_post_processing/Add_Gaussian_Noise.py`

**Comment by GoodJuicee** [comment]
> Please run the preprocessing in this file to generate the dataset: `pre_post_processing/Add_Gaussian_Noise.py`
Thank you for your reply, would you like to ask Add_ Gaussian_ The purpose of Noise. py is to add noise. Why do we add noise? Shouldn't we downsample the dataset? Or should we downsample the 8i dataset first and then add noise
And there are also four datasets in the 8iVFBv2 dataset, including longpress. root. redandblack. oldier. How to divide the training set, testing set, and validation set, by placing them in a folder and setting the num_test in the following code

parser.add_argument("--dataset", default='../Data/8iVFBv2/')
  parser.add_argument("--downsample", default=8, help='Downsample Rate')
  parser.add_argument("--num_test", type=int, default=1024, help='how many of the dataset use for testing')
  parser.add_argument("--dataset_8i", default='../Data/8i_test/8x/')
  parser.add_argument("--dataset_8i_GT", default='../Data/8i_test/orig/')
